app/views/pages/components_dropdowns.html.erb

```diff
@@ -0,0 +1,53 @@
+
+<%# ------dropdown filter-------- %>
+
+<ul class="dropdown-filter">
+  <li class="dropdown">
+    <button type="button" class="btn" data-bs-toggle="dropdown" class="dropdown-toggle">
+      Duration<b class="filter-title"></b>
+    </button>
+    <ul class="dropdown-menu">
+      <li><label class="checkbox"><input type="checkbox" class="checkbox-text">    15 min</label></li>
+      <li><label class="checkbox"><input type="checkbox" class="checkbox-text">    30 min</label></li>
+      <li><label class="checkbox"><input type="checkbox" class="checkbox-text">    1 heure</label></li>
+      <li><label class="checkbox"><input type="checkbox" class="checkbox-text">    2 heures</label></li>
+    </ul>
+  </li>
+</ul>
+
+<%# ------dropdown pour signaler un hotspot-------- %>
+
+<div class="btn-group" id="dropdown-hotspots">
+  <button type="button" class="btn btn-danger" data-bs-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+    <i class="fa-solid fa-thumbtack"></i>
+  </button>
+  <div class="dropdown-menu">
+    <a class="dropdown-item" href="#">Dog park</a>
+    <a class="dropdown-item" href="#">Fountain</a>
+    <div class="dropdown-divider"></div>
+    <a class="dropdown-item" href="#">Distributor</a>
```

Comment:
Dans la DB c'est noté en dispenser si je dis pas de bêtise

Reply:
+ indentation

Reply:
Ah oui c'est vrai. Je corrige.